src/pages/HomePage/views/PublicPoolView.vue

```diff
@@ -163,7 +164,11 @@ const barIndicators = computed<InfoBarType.Indicator[]>(() => [
   },
   {
     title: t('home-page.public-pool-view.withdraw-at-title'),
-    value: poolData.value
+    value: userPoolData.value?.claimLockEnd
+      ? new Time(userPoolData.value?.claimLockEnd.toNumber()).format(
+          DEFAULT_TIME_FORMAT,
+        )
+      : poolData.value
```

Comment:
Too much nested ternary operators, makes it hard to read.

Reply:
done